ovs/proto_trace.go

```diff
@@ -34,6 +34,12 @@ var (
 	traceStartRegexp      = regexp.MustCompile(`bridge\("(.*)"\)`)
 	traceFlowRegexp       = regexp.MustCompile(` *[[:digit:]]+[.] ([[:alpha:]].*)`)
 	traceActionRegexp     = regexp.MustCompile(` +([[:alpha:]].*)`)
+	recircIDRegexp        = regexp.MustCompile(`recirc_id=`)
```

Comment:
Does this not save the recirc id? I was under the impression that it was needed to tie different traces together. I recall seeing some runs where there were 3 traces and the only way I was comfortable knowing the order they came in was to match the recirc_id from the `Datapath Actions` line with the recirc id at the beginning of one of the following traces.